src/type.js

```diff
@@ -55,7 +55,11 @@ ChartInternal.prototype.isBarType = function (d) {
 };
 ChartInternal.prototype.isScatterType = function (d) {
     var id = isString(d) ? d : d.id;
-    return this.config.data_types[id] === 'scatter';
+    return this.config.data_types[id] === 'scatter' || this.config.data_types[id] === 'stanford';
```

Comment:
I think this is confusing. How about creating another method `isScatterOrStanfordType` and replace the usages of `isScatterType` with it where necessary.

Reply:
@kt3k I did what you suggested, see if the changes I made work for you. Thanks!